spec/electra/beaconblockbody.go

```diff
@@ -24,19 +24,21 @@ import (
 )
 
 // BeaconBlockBody represents the body of a beacon block.
+//
+//nolint:revive
 type BeaconBlockBody struct {
 	RANDAOReveal          phase0.BLSSignature `ssz-size:"96"`
 	ETH1Data              *phase0.ETH1Data
 	Graffiti              [32]byte                      `ssz-size:"32"`
-	ProposerSlashings     []*phase0.ProposerSlashing    `ssz-max:"16"`
-	AttesterSlashings     []*AttesterSlashing           `ssz-max:"1"`
-	Attestations          []*Attestation                `ssz-max:"8"`
-	Deposits              []*phase0.Deposit             `ssz-max:"16"`
-	VoluntaryExits        []*phase0.SignedVoluntaryExit `ssz-max:"16"`
+	ProposerSlashings     []*phase0.ProposerSlashing    `dynssz-max:"MAX_PROPOSER_SLASHINGS" ssz-max:"16"`
+	AttesterSlashings     []*AttesterSlashing           `dynssz-max:"MAX_ATTESTER_SLASHINGS" ssz-max:"1"`
+	Attestations          []*Attestation                `dynssz-max:"MAX_ATTESTATIONS"       ssz-max:"8"`
```

Comment:
Shouldn't this be `MAX_ATTESTATIONS_ELECTRA`?

Reply:
that actually solved an issue that I was looking for since quite some time :D  thx :)